.envrc

```diff
@@ -1 +1,5 @@
 export PATH=${PATH}:./target/debug
+export BASE_FEE_RATE=0.005
```

Comment:
This is not where to document these values.

We only want to use env vars to inject sensitive data. Everything else goes into `stablesats.yml` - though these are defaults that should be hard coded in the code.